### DISCUSSION TITLE
更换适配器 bot登陆出现 nonebot | OneBot V12 | Missing connect meta event

### DISCUSSION THREAD

**Discussion by dongmingzjj** [nonebot/discussions#491] [问答]
新增了入口文件
```python
import nonebot
from nonebot.adapters.onebot.v12 import Adapter

nonebot.init()

driver = nonebot.get_driver()
driver.register_adapter(Adapter)

if __name__ == "__main__":
    nonebot.load_builtin_plugins()
    nonebot.load_plugins("plugins")
    nonebot.run()
```
其他就是将v11更换成了v12
v11 下是正常运行的 
![image](https://github.com/user-attachments/assets/a3feb532-9cfd-4be3-858e-bc070059f5c8)
  
后测试更换v12 为 v11 也可运行

连接的是LLOneBot

**Comment by yanyongyu** [comment]
有没有一种可能 v11 和 v12 是两种协议，而你使用的llob是v11，并不能使用 v12 连接

**Comment by dongmingzjj** [comment]
好的👌，感谢，我试试


---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月16日(周五) 晚上8:39
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [nonebot/discussions]更换适配器 bot登陆出现




 
有没有一种可能 v11 和 v12 是两种协议，而你使用的llob是v11，并不能使用 v12 连接
 
&mdash;
